lib/editor/components/HourMinuteInput.js

```diff
@@ -67,11 +68,13 @@ export default class HourMinuteInput extends Component<Props, State> {
   }
 
   render () {
-    const {seconds: initialSeconds, style, ...otherProps} = this.props
+    const {seconds: initialSeconds, standaloneInput, style, ...otherProps} = this.props
     const {hours, minutes} = this.state
 
     const defaultValue = !minutes && !hours ? '' : '00'
 
+    const roundLeftBorder = '4px 0px 0px 4px'
+    const roundRightBorder = '0px 4px 4px 0px'
```

Comment:
I would do the border decision-making here instead of down below. Use a template string and set the 4/0 based on `standaloneInput`. Also where are the border radii being set at the moment?